pgmpy/factors/CPD.py

```diff
@@ -112,6 +112,27 @@ def __init__(self, variable, variable_card, values,
 
         super().__init__(variables, cardinality, values.flatten('C'))
 
+    def replace_variables(self, var1, var2):
```

Comment:
Have an additional input argument ```inplace```; when set as ```inplace=True``` it should modify the existing CPD else should return a new modified CPD.